app/models/manageiq/providers/embedded_ansible/provider.rb

```diff
@@ -9,8 +7,13 @@ class ManageIQ::Providers::EmbeddedAnsible::Provider < ::Provider
           :dependent   => :destroy, # to be removed after ansible_tower side code is updated
           :autosave    => true
 
-  def self.raw_connect(base_url, username, password, verify_ssl)
-    return super if MiqRegion.my_region.role_active?('embedded_ansible')
-    raise StandardError, 'Embedded ansible is disabled'
+  before_validation :ensure_managers
```

Comment:
Note:  Added this which originated from the shared code from the tower provider.  Allowed the delete spec for `provider_spec.rb` to function as it would without needing to change to much.

This could have been a stupid addition though, so let me know if this shouldn't be here.

Reply:
It's not likely that it will hurt anything so I think it should be fine.

Reply:
Oh good call...I didn't realize this was the structure. I did it manually during seeding, so can you just remove the seeding code that is effectively duplicating this?  https://github.com/ManageIQ/manageiq/pull/18687/files#diff-ecbace4ce3e69bc520a0c27e93194918R11

Reply:
> so can you just remove the seeding code that is effectively duplicating this?

Oh nice, yeah, I will do that.

Reply:
Marking this as resolved since I will be pushing the requested changed shortly (but it won't be reflected here).